db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :ip_or_hostname
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devs, :array => true, :default => []
```

Comment:
yeah, and I'm all in for using native support in psql. I'm just not so sure, because this would be the first time we do this and I know we do db replication on db level, thats why I pulled @carbonin  in here.